### PR TITLE
Fixed CircleCi. Added missing shared target schemes.

### PR DESCRIPTION
**Pull request by Elland**
✨

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -7,11 +7,11 @@ test:
   override:
     - xcodebuild
         -workspace Toshi.xcworkspace
-        -scheme "All Tests"
```
Comment (yuliaveres) [inline circle.yml]: i remember it was changed to this not long time ago ? was that the reason of current Circle CI issues ?

Comment (Elland) [reply circle.yml]: Yes. There's no such scheme. Schemes were not being synched through `git` and circle was pointing to one that didn't even exist locally. 😅

Comment (marijnschilling) [reply circle.yml]: How is it fixed now then? Did you need to manually add the schemes to git?

Comment (Elland) [reply circle.yml]: Precisely. Added the `.xcscheme` files and set the correct testable target [here](https://github.com/toshiapp/toshi-ios-client/pull/578/commits/ad11e1da3280f19671e1e4c513705b494aad159d#diff-f76ec1f905240c5183ec1b2f7bf20811R25).

**Comment by marijnschilling** [comment]
Cool, did you double check with committing a failing tests? 😬

**Comment by Elland** [comment]
@marijnschilling check it now.

**Comment by Elland** [comment]
@marijnschilling CI being a bit weird. Showing all tests passing, but if you go inside it and inspect, it shows the failed test. 👀

**Comment by Elland** [comment]
@yuliaveres wanna double check?

**Comment by yuliaveres** [comment]
looks ok!

**Pull request merged**